### PR TITLE
cleanup: remove episode user record contracts from the db

### PR DESCRIPTION
**Pull request by revam**
remove episode user record contracts from the db and generate the client user contracts on the fly instead.

### CODE REVIEW

File: Shoko.Server/Databases/DatabaseFixes.cs [hunk 1]
```diff
@@ -805,7 +805,7 @@ public static void FixWatchDates()
             // Fetch every episode user record stored to both remove orphaned records and to make sure the watch date is correct.
             var userDict = RepoFactory.JMMUser.GetAll().ToDictionary(user => user.JMMUserID);
             var fileListDict = RepoFactory.AnimeEpisode.GetAll().ToDictionary(episode => episode.AnimeEpisodeID, episode => episode.GetVideoLocals());
-            var episodesURsToSave = new List<SVR_AnimeEpisode_User>();
+            var episodeURsToSave = new List<SVR_AnimeEpisode_User>();
```
Comment (Cazzar) [inline Shoko.Server/Databases/DatabaseFixes.cs:808]: Refactor typo?

File: Shoko.Server/Models/SVR_AnimeEpisode.cs [hunk 2]
```diff
@@ -146,7 +146,39 @@ public List<CL_VideoDetailed> GetVideoDetailedContracts(int userID)
...
         public CL_AnimeEpisode_User GetUserContract(int userID, ISessionWrapper session = null)
         {
-            return GetOrCreateUserRecord(userID, session).Contract;
+            var anidbEpisode = AniDB_Episode;
+            var seriesUserRecord = RepoFactory.AnimeSeries_User.GetByUserAndSeriesID(userID, AnimeSeriesID);
+            var episodeUserRecord = GetOrCreateUserRecord(userID, session);
+            var englishTitle = RepoFactory.AniDB_Episode_Title.GetByEpisodeIDAndLanguage(AniDB_EpisodeID, "EN")
+                .FirstOrDefault()?.Title;
+            var romajiTitle = RepoFactory.AniDB_Episode_Title.GetByEpisodeIDAndLanguage(AniDB_EpisodeID, "X-JAT")
+                .FirstOrDefault()?.Title;
```
Comment (Cazzar) [inline Shoko.Server/Models/SVR_AnimeEpisode.cs:155]: Best to inline where you can, and it looks like the last two aren't used.

**Review by Cazzar — CHANGES_REQUESTED** [review]

File: Shoko.Server/Models/SVR_AnimeEpisode.cs (hunk 2 above)
Comment (revam) [reply Shoko.Server/Models/SVR_AnimeEpisode.cs:155]: Forgot to remove them :sweat_smile:

**Comment by revam** [comment]
Will do the database migrations tomorrow.

**Comment by revam** [comment]
This is now blocked by https://github.com/ShokoAnime/ShokoServer/pull/951

**Comment by revam** [comment]
Had to update SQLite to use the DROP COLUMN syntax

**Pull request merged**